### DISCUSSION TITLE
App Control Messages

### DISCUSSION THREAD

**Discussion by ZilogZ80a** [d4rken-org/sdmaid-se#1519] [Q&A]
Whenever App Control is executed, it comes up with what I assume are humorous messages while the system readies the data.  When I saw "waiting for Pepe Sylvia" I became concerned.  Is this in fact a humorous message displayed instead of "Please wait" or is it a sign of malware?  Not funny.

**Comment by d4rken** [comment]
> Is this in fact a humorous message displayed instead of "Please wait"

Yes.

It's a reference to https://knowyourmeme.com/memes/pepe-silvia

**Comment by ZilogZ80a** [comment]
OK :-)
-----Original Message-----
From: Matthias Urhahn ***@***.***>
Reply-To: d4rken-org/sdmaid-se
***@***.***>
To: d4rken-org/sdmaid-se ***@***.***>
Cc: ZilogZ80a ***@***.***>, Author
***@***.***>
Subject: Re: [d4rken-org/sdmaid-se] App Control Messages (Discussion
#1519)
Date: Thu, 26 Dec 2024 00:15:39 -0800

> Is this in fact a humorous message displayed instead of "Please wait"
Yes.
It's a reference to https://knowyourmeme.com/memes/pepe-silvia
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID:
***@***.***
>
